src/swapd/wallet.rs

```diff
@@ -69,9 +69,11 @@ pub struct HandleBuyProcedureSignatureRes {
     pub buy_tx: bitcoin::Transaction,
 }
 
-#[derive(Clone, Debug, StrictEncode, StrictDecode)]
+#[derive(Clone, Display, Debug, StrictEncode, StrictDecode)]
 pub enum Wallet {
+    #[display("Wallet::Alice")]
```

Comment:
Maybe a more human readable form for display would be good? e.g. `Alice wallet`

Reply:
1d1e45bd81f003cfcbec79987954ae7647e79fd5